src/olympia/users/models.py

```diff
@@ -962,6 +962,9 @@ def watch_changes(old_attr=None, new_attr=None, instance=None,
     )
     if any(field in changes for field in basket_relevant_changes):
         from olympia.amo.tasks import sync_object_to_basket
+        log.info(
+            'Triggering a sync of %s %s with basket because of %s change',
+            'userprofile', instance.pk, 'attribute')
```

Comment:
Why the unnecessary string substitution in each `log.info`?

Reply:
Just to make sure the string was consistent without having to write a wrapper for it

Reply:
you could have specified the string as a constant.